Dockerfile

```diff
@@ -175,7 +175,7 @@ RUN apt-get -y remove gcc \
 # CONFIGURE  ENVIRONMENT#
 #########################
 ENV PATH ${CONDA_HOME}/miniconda/envs/tethys/bin:$PATH 
-VOLUME ["${TETHYS_HOME}/workspaces", "${TETHYS_HOME}/keys"]
+VOLUME ["${WORKSPACE_ROOT}", "${TETHYS_HOME}/keys"]
```

Comment:
@swainn I was going to ask you why we have this specific directories mapped as volumes. It seems like we should map the whole `TETHYS_PERSIST` dir.

Reply:
Remind me are these directories in the `TETHYS_PERSIST` directory? If so, then yes. We should do that. I think that's how we configure our k8s deployments anyway.

Reply:
The default `WORKSPCE_ROOT` is in `TETHYS_PERSISIT`.  The `${TETHYS_HOME}/keys` is obviously in `TETHYS_HOME`, but it's not used anywhere.